internal/kubeclient/roundtrip.go

```diff
@@ -146,7 +146,7 @@ func handleOtherVerbs(
 
 	result, err := middlewareReq.mutateRequest(obj)
 	if err != nil {
-		return true, nil, err
+		return true, nil, fmt.Errorf("middleware request for %#v failed to mutate: %w", middlewareReq, err)
```

Comment:
Would `middlewareReq` potentially have any secrets that might get logged due to this error message?

Reply:
It is just metadata:

```go
type request struct {
	verb                Verb
	namespace           string
	resource            schema.GroupVersionResource
	reqFuncs, respFuncs []func(obj Object) error
	subresource         string
}
```

It is a bit noisy (added some newlines):

```
pinniped-concierge-749966bbf4-ff48k pinniped-concierge E0820 14:09:51.701071       1 controller.go:220]
legacy-pod-cleaner-controller: {concierge test-legacy-kube-cert-agent-dh2kx} failed with:
could not delete legacy agent pod:
Delete "https://10.96.0.1:443/api/v1/namespaces/concierge/pods/test-legacy-kube-cert-agent-dh2kx":
middleware request for
&kubeclient.request{verb:"delete", namespace:"concierge", resource:schema.GroupVersionResource{Group:"", Version:"v1", Resource:"pods"}, reqFuncs:[]func(kubeclient.Object) error{(func(kubeclient.Object) error)(0x1cc22c0)}, respFuncs:[]func(kubeclient.Object) error{(func(kubeclient.Object) error)(0x1c38580)}, subresource:""}
failed to mutate:
request mutation failed:
write attempt rejected as client is not leader
```